File: smart_canteen_fe/lib/models/login_model.dart
class LoginModel {
  final String username;
  final String password;

  LoginModel({
    required this.username,
    required this.password,
  });

  // factory LoginModel.fromJson(Map<String, dynamic> json) {
  //   return LoginModel(
  //     username: json['username'] ?? '',
  //     password: json['password'] ?? '',
  //   );
  // }

  // Chuyển từ LoginModel sang JSON
  Map<String, dynamic> toJson() {
    return {
      'username': username,
      'password': password,
    };
  }
}

File: Smart_Canteen_BE/Smart_Canteen_BE/Model/LoginModel.cs
using System.ComponentModel.DataAnnotations;

namespace Smart_Canteen_BE.Model
{
    public class LoginModel
    {
        [Required(ErrorMessage = "Tên người dùng phải bắt buộc")]
        public string Username { get; set; } = string.Empty;

        [Required(ErrorMessage = "Mật khẩu phải bắt buộc")]
        public string Password { get; set; } = string.Empty;
    }
}
